Modelica/Mechanics/MultiBody/Frames.mo

```diff
@@ -3076,9 +3076,10 @@ Optionally, residues are returned as well if <code>withResidues=true</code>.
         "Orientation object to rotate frame 1 into frame 2";
     protected
       Real e_x[3](each final unit="1")=if length(n_x) < 1e-10 then {1,0,0} else normalize(n_x);
```

Comment:
The function normalize() uses eps=100*ModelicaServices.Machine.eps to avoid a division by zero. Theoretically, there could be a mess if such eps>1e-10. Could it be reasonable to better set
`if length(n_x) < 1e-10 then {1,0,0} else normalize(n_x, 1e-10)`
or
`if length(n_x) < 100*ModelicaServices.Machine.eps then {1,0,0} else normalize(n_x)`
?

Reply:
in addition. Should not rather the higher level `Modelica.Constants.eps` be used instead of `ModelicaServices.Machine.eps`?

Reply:
normalizes already uses Modelica.Constants.eps.

Reply:
> normalize already uses Modelica.Constants.eps

This is correct, sorry for the confusion. 

Actually, my question rather is whether there shall better be the same numerical value for the if-clause threshold and for "normalize" evaluation. (This should of course be a multiple of Modelica.Constants.eps)

Reply:
I would not change these thresholds now, because existing models might be influence in rare cases. Note, whatever threshold is used, there might be (very rare) cases where this heuristics is not good enough.